packages/react-native/React/Fabric/RCTConversions.h

```diff
@@ -146,7 +150,99 @@ inline UIAccessibilityTraits RCTUIAccessibilityTraitsFromAccessibilityTraits(
   }
   return result;
 };
-#endif // [macOS]
+#else // [macOS
+inline NSAccessibilityRole RCTUIAccessibilityRoleFromAccessibilityTraits(
+    facebook::react::AccessibilityTraits accessibilityTraits)
+{
+  using AccessibilityTraits = facebook::react::AccessibilityTraits;
```

Comment:
We should probably align to the w3c-compliant roles that @falselobster updated our paper implementation to:
https://github.com/microsoft/react-native-macos/pull/2101